lib/honeycomb-beeline.rb

```diff
@@ -25,8 +25,8 @@ class << self
     extend Forwardable
     attr_reader :client
 
-    def_delegators :@client, :start_span, :add_field, :add_field_to_trace,
-                   :current_span, :current_trace
+    def_delegators :@client, :libhoney, :start_span, :add_field,
+                   :add_field_to_trace, :current_span, :current_trace
```

Comment:
maybe we could add a basic test to this file for this new delegator? https://github.com/honeycombio/beeline-ruby/blob/master/spec/honeycomb/beeline_spec.rb

Reply:
Good idea - done!